pkg/cmd/roachtest/roachtestutil/mixedversion/mixedversion.go

```diff
@@ -608,7 +608,7 @@ func (t *Test) plan() (*TestPlan, error) {
 	initialRelease := previousReleases[0]
 	planner := testPlanner{
 		versions:       append(previousReleases, clusterupgrade.CurrentVersion()),
-		currentContext: newInitialContext(initialRelease, t.crdbNodes),
+		currentContext: newInitialContext(initialRelease, t.crdbNodes, nil /* tenant */),
```

Comment:
Nice! I assume this is where tenant's nodes will be plugged in a subsequent PR.

Reply:
Correct 👍